code/modules/mob/living/carbon/carbon_defines.dm

```diff
@@ -17,6 +17,7 @@
 	var/pose = null
 	var/list/chem_effects = list()
 	var/intoxication = 0//Units of alcohol in their system
+	var/bac = 0 //Blood alcohol level, as calculated inside intoxication.dm

```

Comment:
Oh nonono. No. Delete this and replace with a proc.

Retaining this state will lead to horrible horrible things. A stateless calculation each time it's needed is better. Also, the name is absoloutely horrible.

Also consider making the proc a `human` proc.